Platforms/QemuQ35Pkg/PlatformBuild.py

```diff
@@ -197,21 +197,10 @@ def __init__(self):
 
     def AddCommandLineOptions(self, parserObj):
         ''' Add command line options to the argparser '''
-
-        # In an effort to support common server based builds this parameter is added.  It is
-        # checked for correctness but is never uses as this platform only supports a single set of
-        # architectures.
-        parserObj.add_argument('-a', "--arch", dest="build_arch", type=str, default="IA32,X64",
-            help="Optional - CSV of architecture to build.  IA32,X64 will use IA32 for PEI and "
-            "X64 for DXE and is the only valid option for this platform.")
-
         CommonPlatform.add_common_command_line_options(parserObj)

```

Comment:
There is some asymmetry between Q35 and SBSA on this method. Is this intentional?

Reply:
Yes. That was added in Q35 mainly for CodeQL which isn't supported in QemuSbsaPkg (due to lack of VS support).